package.json

```diff
@@ -57,6 +57,7 @@
     "lint-staged": "14.0.1",
     "next": "^13.5.2",
     "next-mdx-remote": "^4.4.1",
+    "next-router-mock": "^0.9.10",
```

Comment:
Please don't add a new package just for mocking a package. You can easily mock the next router with a simple object.

Read the Collaborator Guidelines, we recommend avoiding any package as much as possible.

Also I believe this one can be removed, because I still believe that unit test is not necessary.